pkg/security/probe/dentry_resolver.go

```diff
@@ -605,15 +687,28 @@ func (dr *DentryResolver) resolveParentFromMap(mountID uint32, inode uint64, pat
 	return path.Parent.MountID, path.Parent.Inode, nil
 }
 
-// GetParent - Return the parent mount_id/inode
+// GetParent returns the parent mount_id/inode
 func (dr *DentryResolver) GetParent(mountID uint32, inode uint64, pathID uint32) (uint32, uint64, error) {
 	parentMountID, parentInode, err := dr.resolveParentFromCache(mountID, inode)
-	if err != nil {
+	if err != nil && dr.erpcEnabled {
+		parentMountID, parentInode, err = dr.resolveParentFromERPC(mountID, inode, pathID)
+	}
+	if err != nil && err != errTruncatedParentsERPC && dr.mapEnabled {
 		parentMountID, parentInode, err = dr.resolveParentFromMap(mountID, inode, pathID)
 	}
 	return parentMountID, parentInode, err
 }
 
+// BumpCacheGenerations bumps the generations of all the mount points
+func (dr *DentryResolver) BumpCacheGenerations() {
+	dr.cacheGenerationsLock.Lock()
+	defer dr.cacheGenerationsLock.Unlock()
+
+	for genMountID := range dr.cacheGenerations {
```

Comment:
we could just use a unique generation variable as you bump all of them

Reply:
Done